### ISSUE TITLE
AWS 

### ISSUE THREAD

**Issue by cocoy** [oraclesorg/test-templates#20]
Hmm can this run on AWS?

**Comment by cocoy** [comment]
Hmm, thinking of  making this into Ansible deployment playbook :)

**Comment by igorbarinov** [comment]
@cocoy yes, we have plans but have no hands. Would be great to have AWS. 
Ansible playbook could be great too. Would you like to chat more about it? I'm @igorbarinov on Telegram

**Comment by cocoy** [comment]
Hey Igor, ill contact you on telegram. Hope I can help on this and on our own project.
